tests/ttnn/integration_tests/yolov4/test_ttnn_downsample4.py

```diff
@@ -0,0 +1,159 @@
+            )
+
+            ttnn_module_args.c3["use_shallow_conv_variant"] = False
+            ttnn_module_args.c3["weights_dtype"] = ttnn.bfloat8_b
+            conv3_weight, conv3_bias = fold_batch_norm2d_into_conv2d(model.c3, model.b3)
+            update_ttnn_module_args(ttnn_module_args.c3)
+            parameters["c3"], c3_parallel_config = preprocess_conv2d(
+                conv3_weight, conv3_bias, ttnn_module_args.c3, return_parallel_config=True
+            )
+
+            parameters["res"] = {}
+            for i, block in enumerate(model.res.module_list):
+                conv1 = block[0]
+                bn1 = block[1]
+                conv2 = block[3]
+                bn2 = block[4]
+
+                ttnn_module_args["res"][f"resblock_{i}_conv1"] = ttnn_module_args["res"]["0"]
+                ttnn_module_args["res"][f"resblock_{i}_conv1"]["weights_dtype"] = ttnn.bfloat8_b
+                weight1, bias1 = fold_batch_norm2d_into_conv2d(conv1, bn1)
+                update_ttnn_module_args(ttnn_module_args["res"][f"resblock_{i}_conv1"])
+                parameters["res"][f"resblock_{i}_conv1"], _ = preprocess_conv2d(
+                    weight1, bias1, ttnn_module_args["res"][f"resblock_{i}_conv1"], return_parallel_config=True
+                )
+
+                ttnn_module_args["res"][f"resblock_{i}_conv2"] = ttnn_module_args["res"]["3"]
+                ttnn_module_args["res"][f"resblock_{i}_conv2"]["weights_dtype"] = ttnn.bfloat8_b
+                weight2, bias2 = fold_batch_norm2d_into_conv2d(conv2, bn2)
+                update_ttnn_module_args(ttnn_module_args["res"][f"resblock_{i}_conv2"])
+                parameters["res"][f"resblock_{i}_conv2"], _ = preprocess_conv2d(
+                    weight2, bias2, ttnn_module_args["res"][f"resblock_{i}_conv2"], return_parallel_config=True
+                )
+
+            ttnn_module_args.c4["use_shallow_conv_variant"] = False
+            ttnn_module_args.c4["weights_dtype"] = ttnn.bfloat8_b
+            conv4_weight, conv4_bias = fold_batch_norm2d_into_conv2d(model.c4, model.b4)
+            update_ttnn_module_args(ttnn_module_args.c4)
+            parameters["c4"], c4_parallel_config = preprocess_conv2d(
+                conv4_weight, conv4_bias, ttnn_module_args.c4, return_parallel_config=True
+            )
+
+            ttnn_module_args.c5["use_shallow_conv_variant"] = False
+            ttnn_module_args.c5["weights_dtype"] = ttnn.bfloat8_b
+            conv5_weight, conv5_bias = fold_batch_norm2d_into_conv2d(model.c5, model.b5)
+            update_ttnn_module_args(ttnn_module_args.c5)
+            parameters["c5"], c5_parallel_config = preprocess_conv2d(
+                conv5_weight, conv5_bias, ttnn_module_args.c5, return_parallel_config=True
+            )
+
+        return parameters
+
+    return custom_preprocessor
+
+
+@skip_for_wormhole_b0()
+def test_downsample4(reset_seeds):
```

Comment:
This is the expected way for the unit tests: 

`def test_downsample4(device, reset_seeds):`
remove this line `device = ttnn.open_device(device_id=2)`

I know you are sharing the same machine and using different devices/device IDs. That' ok but for the PR please make my suggested changes.